public/class-admwswp-public.php

```diff
@@ -326,8 +329,11 @@ public static function weblinkWidget($attr)
         $html .= "});";
         $html .= "clearInterval(weblinkInterval$widgetId); } }, 500);"; // interval ending
         $html .= "</script>";
-
-        return $html;
+        
+        if ($showAddToCart) {
+            return $html;
+        }
+        return '';
```

Comment:
are we not returning no weblink embedding html at all if the `showAddToCart` is false?

Reply:
Yes this will force not to render the Cart and Basket widgets and this option is only used for Cart and Basket widget type, so by default Im setting it to true, then if the widget type I'm overriding its value to what ever is set from the backed.
in a following ticket will be adding the logic to remove the add to cart button from other widgets: https://administrate.atlassian.net/browse/PLAT-14776